index.js

```diff
@@ -177,15 +175,6 @@ Client.prototype.sync = function (callback) {
     } catch (ex) {}
   }
 
-  if (self.state.updatesStamp < now) {
-    return self._updateRules(function (err) {
-      if (err) self._log('_updateRules', { message: err.toString() })
-
-      self._log('sync', { delayTime: msecs.minute })
-      callback(null, self.state, msecs.minute)
```

Comment:
I don't think we get in here at all. Will confirm

Reply:
Scratch that. We do hit that during contribution after Tx's come in (before the votes)